packages/layout/src/list.tsx

```diff
@@ -27,41 +31,49 @@ interface ListOptions {
   spacing?: SystemProps["margin"]
 }
 
-export interface ListProps extends HTMLChakraProps<"ul">, ListOptions {}
+export interface ListProps
+  extends HTMLChakraProps<"ul">,
+    ThemingProps,
+    ListOptions {}
 
 /**
  * List is used to display list items, it renders a `<ul>` by default.
  *
  * @see Docs https://chakra-ui.com/components/list
  */
 export const List = forwardRef<ListProps, "ul">(function List(props, ref) {
+  const styles = useMultiStyleConfig("List", props)
   const {
     children,
     styleType = "none",
```

Comment:
Please see "default styles" comment

Reply:
`styleType` is a prop, I wasn't able to get it to work with the theme as a default value. Let me know if you have any ideas.

Reply:
Oh, it is an alias for `listStyleType` - missed that 😓